intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -773,19 +773,37 @@ \subsection{When will my job start?}
 
 The \hpcInfra clusters use a fair-share scheduling policy (see \autoref{ch:hpc-policies}). There is no guarantee on when a
 job will start, since it depends on a number of factors. One of these factors is
-the priority of the job, which is determined by
+the priority of the job, which is determined by:
 \begin{itemize}
-    \item historical use: the aim is to balance usage over users, so
+    \item Historical use: the aim is to balance usage over users, so
         infrequent (in terms of total compute time used) users get a higher priority
 
-    \item requested resources (amount of cores, walltime, memory, \ldots)
+    \item Requested resources (amount of cores, walltime, memory, \ldots).
+        The more resources you request, the more likely it is the job(s) will
+        have to wait for a while until those resources become available.
 
-    \item time waiting in queue: queued jobs get a higher priority over time
+    \item Time waiting in queue: queued jobs get a higher priority over time.
 
-    \item user limits: this avoids having a single user use the entire cluster.
+    \item User limits: this avoids having a single user use the entire cluster.
```

Comment:
Technically, this is no longer correct, and if we want it back, we can, but we should decide on it then :)